tests/e2e/utils.js

```diff
@@ -431,3 +431,19 @@ export const createCoupon = async ( coupon ) => {
 
 	return createdCoupon;
 };
+
+/**
+ * Open the block editor settings menu.
+ */
+export const openBlockEditorSettings = async () => {
+	const buttonSelector =
+		'.edit-site-header__actions button[aria-label="Settings"]';
+
+	const isButtonPressed = await page.$eval( buttonSelector, ( el ) =>
+		el.getAttribute( 'aria-pressed' )
+	);
+
+	if ( isButtonPressed === 'false' ) {
+		await page.click( buttonSelector );
+	}
+};
```

Comment:
wouldn't it be cleaner here to wait for the settings drawer/popup/sidebar to be present rather than checking the state of a button?